src/main/java/scraping/GetRatings.java

```diff
@@ -132,20 +132,20 @@ private static String parseInstructorName(String name) {
           }
 
           return new Rating(id, Integer.parseInt(url),
-                            parseRating(resp.getResponseBody()));
+                            parseRating(resp.getResponseBody(), id));
         });
   }
 
-  private static Float parseRating(String rawData) {
+  private static Float parseRating(String rawData, int id) {
```

Comment:
Could you change the return type of this method to primitive float? And then make sure the rating object uses the primitive type instead of the object type?